ramls/instance.json

```diff
@@ -215,6 +215,25 @@
             "type": "string",
             "description": "Date (year YYYY) of publication, distribution, etc."
           },
+          "publicationYear": {
+            "type": "integer",
+            "description": "Publication year"
+          },
+          "publicationPeriod": {
+            "type": "object",
+            "description": "Publication period",
+            "properties": {
+              "start": {
+                "type": "integer",
+                "description": "Publication start year"
+              },
+              "end": {
+                "type": "integer",
+                "description": "Publication end year"
+              }
+            },
+            "additionalProperties": false
+          },
```

Comment:
My interpretation of the JIRA issue is that there is only a single new field intended, that could have different values depending upon if there is a single date or a range, rather than multiple new fields. 

Have these new fields been confirmed with Magda?

Reply:
Yes, it is possible to have a specific year or a period.

Reply:
Ok, that seemed to me to be really quite different from what the issue originally asked for. I think I'm likely missing a lot of context and likely shouldn't have got involved at all (my concern was we were adding properties that wouldn't be used).

I'll reply to Magda's comment on the issue to clarify my understanding and then leave the Falcon team to it. Apologies for barging in :-(